### ISSUE TITLE
Add WP.org review notice as we have for Neve and other products

### ISSUE THREAD

**Issue by mghenciu** [Codeinwp/neve-fse#63]
### What problem does this address?

I think this is already implemented in Themeisle SDK and we can easily add it, in all FSE themes.

### What is your proposed solution?

What is different in this case is that we already have some notices inside, **so this one should appear only after those are already dismissed/expired.**

For Neve FSE we have the survey notice, while for other FSE themes - we have the notice promoting Otter blocks.
 
As discussed on Slack, I suggest:
- show the review notice after 3 weeks
- only if the other notices are dismissed/expired (meaning that there won't be any stacking of notices)

### Will this feature require documentation? (Optional)

None

**Comment by HardeepAsrani** [comment]
@mghenciu Which notice in particular are you talking about?

**Comment by mghenciu** [comment]
I don't

**Comment by selul** [comment]
@HardeepAsrani it's about the review notice from the SDK, which I think is already available since we are using the SDK on Neve FSE

**Comment by HardeepAsrani** [comment]
Yes, that's why I was confused. That isn't added by us manually to each product. It's available from the SDK.

It's only available after a the product has been installed for a certain amount of time, that's 100 hours.

If you want to test, you can go to `wp-admin/options.php` in your WP, find the option `neve_fse_install` or whatever the name is, and change that time string to something a week in past and it should appear.

<img width="1677" alt="Screenshot 2023-08-01 at 1 11 31 PM" src="https://github.com/Codeinwp/neve-fse/assets/2649903/6c4fb7c4-bffa-4b66-96c0-cd2f138f4690">

**Comment by mghenciu** [comment]
Ok, thank you for the explanation and I apreciate all the details @HardeepAsrani.
Then I guess we can close this one, since the Review Notices are already included in the SDK, for our FSE themes.

I'll also post an update in the discussion Slack.
